app/javascript/dashboard/i18n/locale/he/inboxMgmt.json

```diff
@@ -127,6 +127,42 @@
           "ERROR_MESSAGE": "We were not able to authenticate Twilio credentials, please try again"
         }
       },
+      "GUPSHUP": {
```

Comment:
Please revert these changes, it would be auto-generated by the system.

Reply:
Cool so just the locale/en is enough?

Reply:
Yes, the rest would autogenerated by Crowdin.